pkg/ocm/kubeletconfig.go

```diff
@@ -77,6 +79,33 @@ func (c *Client) UpdateKubeletConfig(clusterID string, args KubeletConfigArgs) (
 	return response.Body(), nil
 }
 
+func (c *Client) FindKubeletConfigByName(
+	ctx context.Context, clusterId string, name string) (*cmv1.KubeletConfig, bool, error) {
+
+	if name == "" {
+		return nil, false, fmt.Errorf("'name' for the KubeletConfig cannot be empty")
```

Comment:
Is this error enough :thinking: should we also check in the runner prior to finding to ensure a name is passed if a cluster is HCP 
WDYT?

Reply:
Done.